### PR TITLE
Update node-fetch version. Refs #143

### PR DESCRIPTION
**Pull request by matheo**

### CODE REVIEW

File: packages/isomorphic-unfetch/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "main": "index.js",
   "types": "index.d.ts",
   "dependencies": {
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^3.1.0",
```
Comment (lounsbrough) [inline packages/isomorphic-unfetch/package.json:16]: Doesn't this need to be `3.1.1`? https://github.com/node-fetch/node-fetch/releases/tag/v3.1.1

**Comment by lounsbrough** [comment]
I assume there might be breaking changes here, specifically the module has moved to `ESM` only.

File: packages/isomorphic-unfetch/package.json (hunk 1 above)
Comment (robreinhard) [reply packages/isomorphic-unfetch/package.json:16]: @lounsbrough I agree.

Comment (matheo) [reply packages/isomorphic-unfetch/package.json:16]: `^` resolves to any following minor/patch version so we just need a refresh of the lock file

**Comment by developit** [comment]
fixed in #156

**Pull request closed without merging**